attributes/default.rb

```diff
@@ -32,6 +32,8 @@
 default['authconfig']['ldap']['sudo']['basedn'] = ''
 default['authconfig']['ldap']['sudo']['full_refresh'] = '86400'
 default['authconfig']['ldap']['sudo']['smart_refresh'] = '3600'
+default['authconfig']['ldap']['group_member'] = 'memberuid' 
```

Comment:
I'm not sure how well used this cookbook is, but your change here would effectively change the default from uniqueMember to memberuid. I'd rather not break things for anyone...
Could you change the default to be uniqueMember?